scanpy/datasets/__init__.py

```diff
@@ -295,3 +295,107 @@ def pbmc3k_processed() -> AnnData:
         backup_url='https://raw.githubusercontent.com/chanzuckerberg/cellxgene/master/example-dataset/pbmc3k.h5ad',
     )
     return adata
+
+def SGE10x(sample_ID = 'V1_Breast_Cancer_Block_A_Section_1') -> AnnData:
```

Comment:
please don’t use uppercase letters in function names and parameter names.

Only `TypeNames` and `CONSTANTS` have uppercase letters.

Reply:
Can you call the function `read_visium10x_h5`, it's more consistent with scanpy read functions and more specific for the type of data.

Reply:
I changed the function name to visium_sge().